src/renderer/cell.zig

```diff
@@ -237,13 +237,27 @@ pub fn isCovering(cp: u21) bool {
 
 /// Returns true of the codepoint is a "symbol-like" character, which
 /// for now we define as anything in a private use area and anything
-/// in the "dingbats" unicode block.
+/// in several unicode blocks:
+/// - Dingbats
+/// - Emoticons
+/// - Miscellaneous Symbols
+/// - Enclosed Alphanumerics
+/// - Enclosed Alphanumeric Supplement
+/// - Miscellaneous Symbols and Pictographs
+/// - Transport and Map Symbols
 ///
 /// In the future it may be prudent to expand this to encompass more
 /// symbol-like characters, and/or exclude some PUA sections.
 pub fn isSymbol(cp: u21) bool {
+    // TODO: This should probably become a codegen'd LUT
```

Comment:
Oh yeah. This is super nasty. I think we should make a "task" type issue to track this right now. It might be worth making a LUT based on our current unicode stuff before we even get the ziglyph replacement figured out.